### PR TITLE
Fixed language and typos

### PR DESCRIPTION
**Pull request by D3v3sh5ingh**

### CODE REVIEW

File: Reference/Config/umbracoSettings/index.md [hunk 1]
```diff
@@ -11,20 +11,20 @@ Below you can see the settings that affect the Umbraco backoffice (v7.8+)
...
 
-The section is used for controlling the backoffice tours functionality
+The section is used for controlling the backoffice tours functionality.
 
 	<backOffice>
         <tours enable="true"></tours>
     </backOffice>
 
-There is only one supported attribute on the tours element:
+There is only one supported attribute on the tours element :
```
Comment (marcemarc) [inline Reference/Config/umbracoSettings/index.md]: Should there be a space here?

generally in the docs we prefer the colon next to the word and without a space...

Comment (D3v3sh5ingh) [reply Reference/Config/umbracoSettings/index.md]: Yes, got this.
Fixed that.

**Comment by D3v3sh5ingh** [comment]
@marcemarc Please review changes

**Comment by marcemarc** [comment]
Merged... Thanks @D3v3sh5ingh do you have an Our Umbraco profile?

**Comment by D3v3sh5ingh** [comment]
@marcemarc Yes i have created my profile with email id d3v3sh.singh@gmail.com 
You can also contact me on this email for reward and swag.
Thank you!

**Comment by nul800sebastiaan** [comment]
Hi there @D3v3sh5ingh,

First of all: **A big #H5YR** for contributing to Umbraco during Hacktoberfest! We are very thankful for the huge amount of PRs submitted, and all the amazing work you've been doing :pray:

If you haven't heard already we have had a record breaking amount of PR's in both the CMS and the Documentation repository. You can read all about the numbers in the [Umbraco Hacktoberfest Aftermath blog post](https://umbraco.com/blog/the-2018-hacktoberfest-aftermath/).

In the spirit of Hacktoberfest we've prepared some **exclusive Umbraco swag** for all our contributors - _including you_! 

[Receive your swag!](https://goo.gl/forms/smjQwEMDM5cUnz6x1) :point_left: Please follow this link to fill out and submit the form, **before November 21st 2018**.

Following this date we'll be sending out all the swag, which also means that it might not reach your doorstep before December, so please bear with us and be patient :pray:

The only thing left to say is **thank you so much for participating in Hacktoberfest**! We really appreciate the help!

Kind regards,
The Umbraco PR Team & The Documentation Curators

**Pull request merged**